live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java

```diff
@@ -66,4 +66,30 @@ void testDeleteRequest()
                 }""";
         containerSupport.expectHxIngestMessageReceived(expectedBody);
     }
+
+    @Test
+    void testDeleteRequestWithNoResource()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
```

Comment:
Should HxI mock be prepared to return 200 if there shouldn't be any ingestion performed? 🤔

Reply:
No - good spot.